ucte/ucte-converter/src/main/java/com/powsybl/ucte/converter/UcteImporter.java

```diff
@@ -147,6 +158,12 @@ private static void createBuses(UcteNetworkExt ucteNetwork, Network network) {
                 continue;
             }
 
+            ReportBuilder substationReport = new ReportBuilder()
```

Comment:
You should write it inline:
```java
   substationReporter.report(new ReportBuilder()
                .withKey("createSubstation")
                .withDefaultMessage("Create substation ${substationName}")
                .withValue("substationName", ucteSubstation.getName())
                .withSeverity("TRACE")
                .build());
```

Would be better:
```java
   substationReporter.report(Report.builder()
                .withKey("createSubstation")
                .withDefaultMessage("Create substation ${substationName}")
                .withValue("substationName", ucteSubstation.getName())
                .withSeverity("TRACE")
                .build());
```

Reply:
Done, inlined with Report.builder()